### PR TITLE
Improve poh.rs readability

### PR DESCRIPTION
**Pull request by metaproph3t**
#### Summary of Changes
* use a constant for variable hashes per second
* `elapsed` -> `elapsed_ms`

### CODE REVIEW

File: entry/src/poh.rs [hunk 1]
```diff
@@ -92,7 +94,7 @@ impl Poh {
 
         // If the hashes_per_tick is variable (std::u64::MAX) then always generate a tick.
         // Otherwise only tick if there are no remaining hashes
-        if self.hashes_per_tick < std::u64::MAX && self.remaining_hashes != 0 {
+        if self.hashes_per_tick != VARIABLE_HASHES_PER_TICK && self.remaining_hashes != 0 {
```
Comment (metaproph3t) [inline entry/src/poh.rs:97]: These are semantically equivalent because there is no u64 larger than std::u64::MAX

**Comment by metaproph3t** [comment]
@jeffwashington Sir, would it be possible for you to review this PR? I see that you are the more recent contributor as per git blame.

**Comment by bw-solana** [comment]
Hopping on a plane soon, but I can take a look at this when I get a minute

**Comment by metaproph3t** [comment]
@bw-solana Many thanks, sir.

**Comment by metaproph3t** [comment]
@bw-solana Am I looking in the right place, sir? For some reason, no comments show up in my client.

PRNoComments

**Comment by CriesofCarrots** [comment]
Fyi @bw-solana , you have to actually submit the review for those pending comments to show up to others.
Is it worth rejuvenating this?

**Review by bw-solana — APPROVED** [review]
LGTM

**Pull request merged**